trouble starting monerod.exe from windows 10 64-bit cmd window

![monero daemon start message](https://user-images.githubusercontent.com/13257104/27721145-53c7cb1a-5d13-11e7-9948-cf42d0a92ba0.PNG)
![mymonerowallet not connect window](https://user-images.githubusercontent.com/13257104/27721147-53cc0c7a-5d13-11e7-9099-32bbf1f25c12.PNG)
![windows 10 gui mymonerowallet address book window](https://user-images.githubusercontent.com/13257104/27721146-53c9a412-5d13-11e7-8d15-0012aa26e7db.PNG)
![windows 10 gui mymonerowallet daemon failed to start message](https://user-images.githubusercontent.com/13257104/27721150-53d0d264-5d13-11e7-9498-2c05d21f366b.PNG)
![windows 10 gui mymonerowallet history window](https://user-images.githubusercontent.com/13257104/27721149-53cf4390-5d13-11e7-8720-5c952316058c.PNG)
![windows 10 gui mymonerowallet receive window](https://user-images.githubusercontent.com/13257104/27721148-53ccf1da-5d13-11e7-9293-4b592db65bc7.PNG)
![windows 10 gui mymonerowallet send window](https://user-images.githubusercontent.com/13257104/27721152-53e1c100-5d13-11e7-9bfe-85b5655e82cf.PNG)
![windows 10 gui mymonerowallet settings window](https://user-images.githubusercontent.com/13257104/27721151-53df974a-5d13-11e7-934f-afafd9453237.PNG)







I am trying to get monerod started as a node from either the GUI or CLI in Windows, but monerod code is failing.  I have been trying to run it as installed in the Downloads\monero-win-x64-v0.10.3.1\ folder or the Program Files\monero-gui-0.10.3.1-beta2\ folder.

Both fail.

The CLI interface gives me these messages:

C:\Users\Owner\Downloads\monero-win-x64-v0.10.3.1\monero-v0.10.3.1>monerod
2017-06-29 21:12:03.612 14104   INFO    global  contrib/epee/src/mlog.cpp:145   New log categories: *:WARNING,net:FATAL,net.p2p:FATAL,net.cn:FATAL,global:INFO,verify:FATAL,stacktrace:INFO
2017-06-29 21:12:03.612 14104   INFO    global  src/daemon/main.cpp:282 Monero 'Wolfram Warptangent' (v0.10.3.1-release)
2017-06-29 21:12:03.612 14104   INFO    global  src/daemon/protocol.h:55        Initializing cryptonote protocol...
2017-06-29 21:12:03.612 14104   INFO    global  src/daemon/protocol.h:60        Cryptonote protocol initialized OK
2017-06-29 21:12:03.612 14104   INFO    global  src/daemon/p2p.h:63     Initializing p2p server...
2017-06-29 21:12:11.345 14104   INFO    global  src/daemon/p2p.h:68     P2p server initialized OK
2017-06-29 21:12:11.345 14104   INFO    global  src/daemon/rpc.h:58     Initializing core rpc server...
2017-06-29 21:12:11.345 14104   INFO    global  contrib/epee/include/net/http_server_impl_base.h:70  Binding on 127.0.0.1:18081
2017-06-29 21:12:11.345 14104   INFO    global  src/daemon/rpc.h:63     Core rpc server initialized OK on port: 18081
2017-06-29 21:12:11.345 14104   INFO    global  src/daemon/core.h:73    Initializing core...
2017-06-29 21:12:11.345 14104   INFO    global  src/cryptonote_core/cryptonote_core.cpp:326     Loading blockchain from folder C:\ProgramData\bitmonero\lmdb ...
2017-06-29 21:12:11.361 14104   WARN    blockchain.db.lmdb      src/blockchain_db/lmdb/db_lmdb.cpp:71   Failed to query m_blocks: MDB_BAD_TXN: Transaction must abort, has a child, or is invalid
2017-06-29 21:12:11.361 14104   ERROR   cn      src/cryptonote_core/cryptonote_core.cpp:395     Error opening database: Failed to query m_blocks: MDB_BAD_TXN: Transaction must abort, has a child, or is invalid
2017-06-29 21:12:11.361 14104   INFO    global  src/daemon/rpc.h:90     Deinitializing rpc server...
2017-06-29 21:12:11.361 14104   INFO    global  src/daemon/p2p.h:90     Deinitializing p2p...
2017-06-29 21:12:11.377 14104   INFO    global  src/daemon/core.h:89    Deinitializing core...
2017-06-29 21:12:11.377 14104   ERROR   daemon  src/daemon/core.h:94    Failed to deinitialize core...
2017-06-29 21:12:11.377 14104   INFO    global  src/daemon/protocol.h:77        Stopping cryptonote protocol...
2017-06-29 21:12:11.377 14104   INFO    global  src/daemon/protocol.h:81        Cryptonote protocol stopped successfully

C:\Users\Owner\Downloads\monero-win-x64-v0.10.3.1\monero-v0.10.3.1>

The GUI gives me this messsage:

[Use Monero.pdf](https://github.com/monero-project/monero-site/files/1114104/Use.Monero.pdf)
![monerd windows startup failure](https://user-images.githubusercontent.com/13257104/27720814-e8bee648-5d10-11e7-9ea6-45cabc16a8a1.PNG)

--- a/README.md
+++ b/README.md
@@ -44,7 +44,7 @@ If for any reason you have questions or need to contact us, you can find us on t
 
 * GitHub: Pretty much everything in Monero is hosted on [GitHub](https://github.com) and uses Git as the primary version control system. If you're not familiar with how to use Git, you can check out [this tutorial](https://guides.github.com/activities/hello-world/) for a good overview. It will take you through pretty much everything you'll need to know to edit the website. If you haven't already, register for GitHub and fork the [Monero Website repository](https://github.com/monero-project/monero-site).
 
-* Markdown experience: To write pages, you're going to need to know how to use Markdown. It's basically a in-between language that enables people who don't know HTML to just write, and it will be compiled into HTML for you. You can find a great Markdown cheat sheet with examples [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). IF you need more help, Google and YouTube are great resources.
+* Markdown experience: To write pages, you're going to need to know how to use Markdown. It's basically an in-between language that enables people who don't know HTML to just write, and it will be compiled into HTML for you. You can find a great Markdown cheat sheet with examples [here](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet). IF you need more help, Google and YouTube are great resources.
 
 *Note: If you're confused, feel free to click other files in the same directory (folder) that you are in for the step that you are on to see some working examples. Compare them to the instructions and you should understand better.*
 
@@ -58,7 +58,7 @@ If for any reason you have questions or need to contact us, you can find us on t
 
 Once you have the above list of things, it's typically a good idea to build the website from your local computer to make sure it works before you make any changes. To do this, complete the following steps:
 
-1. Navigate to the your local `monero-site` repository.
+1. Navigate to your local `monero-site` repository.
 2. Serve the website: `jekyll serve`
 3. Open a browser and go to [http://127.0.0.1:4000](http://127.0.0.1:4000).
 4. If all went well, you should see the Monero website and you're ready to make changes.
@@ -69,11 +69,20 @@ The average Monero user that will want to contribute to the website should proba
 
 If you are a web developer and would like to make large macro-level changes, it would be best to get in contact with rehrar. The HTML/CSS framework is custom, though quite easy to use for a seasoned developer. Still, there are some pages that are more complicated than others and would require fair amounts of styling changes should the structure change significantly.
 
-This website is completely open-source however, and anything and everything is available for changing should the community deem it necessary.
+This website is completely open-source however and anything and everything is available for changing should the community deem it necessary.
 
 Every section from here on out will talk about how to make a specific type of web page. It will start with a bullet point list of what to do for the advanced among you that just want a quick overview. For those who are still learning this list is followed by a detailed explanation, starting with example front matter. Any variable in the front matter written in all caps you are expected to change (make sure your changes are not all caps though). It will then lead you through the rest of the process until it's time to type your content.
 
 All external links must have http:// or https:// in front of them or they will not redirect properly.
+
+### 3.1 Housekeeping
+#### GitHub Issues
+We ask that if you open an issue on the site that you remain available for clarifying questions or corrections. We do our best to close issues that are resolved when we make changes to the site, but If your issue is resolved by a contributer and the issue is not closed we ask that you close it in a timely manner. A contributer (rerhar, SamsungGalaxyPlayer, or other community members, etc) may ask you to close an issue after it's confirmed fixed. Please review the changes to the site and close your issue if you can verify that it's fixed.
+
+#### Pull Requests
+Contributers should use [issue keywords](https://help.github.com/articles/closing-issues-using-keywords/) to make it easier for maintainers to close issues when they merge. Include close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved, etc in the commit message or pull request description, so that the correct issue can be closed if your PR is merged. (Example: 'fixes #1234' could close Issue 1234 when merged.)
+
+Pull requests allow others to make comments or review your changes to the site. We ask that you remain available to comment or make changes to your PR. Pull requests with pending changes for more than 30 days will be closed and need to be resubmitted in the future. Sometimes someone else's changes might make your changes conflict with the current site. If that happens you may need to rebase your PR. (If you're unsure about how to do so, you can reach out to other contributers on IRC (freenode #monero) and someone should be able to walk you through it.
 
 ## 4.0 How to make a blog post
 
@@ -111,7 +120,7 @@ You're all done. Submit a PR and wait for it to be reviewed and merged. Be sure 
 
 ### 5.1 Quick Start
 
-* Make file in /resources/user-guides with a .md ending and no spaces in filename.
+* Make file in /resources/user-guides with an .md ending and no spaces in filename.
 * Front Matter as in 5.3
 * Write User Guide
 * Add guide using markdown in the correct category in /resources/user-guides/index.md being careful not to mess with any indentation
@@ -139,7 +148,7 @@ DO NOT CHANGE ANYTHING IN THIS DOCUMENT BESIDES WHAT YOU ARE INSTRUCTED TO.
 
 This file will look quite different because it's HTML. Don't panic. Simply Ctrl + F (i.e. the find feature) and search for the category that you want to put your User Guide in. You will see there are some sections that are not indented like the others. They are flush with the left side of the screen. **Do not change the indentation.** You can put markdown in these areas.
 
-Once you've identified the non-indented area under the category you would like your User Guide to be under, you can use markdown to insert your link underneath the others. `[TITLE OF USER GUIDE](LINK-TO-USER-GUIDE.html)`. Please note that the file name in between the paranthases must be EXACTLY the same name as the file name you made in step 5.2, but with a `.html` at the end instead of `.md`.
+Once you've identified the non-indented area under the category you would like your User Guide to be under, you can use markdown to insert your link underneath the others. `[TITLE OF USER GUIDE](LINK-TO-USER-GUIDE.html)`. Please note that the file name in between the parentheses must be EXACTLY the same name as the file name you made in step 5.2, but with a `.html` at the end instead of `.md`.
 
 In the event that you think your User Guide should be in a new Category that doesn't exist yet, contact rehrar to make one for you.
 
@@ -296,9 +305,9 @@ and paste it IN THE CHRONOLOGICAL ORDER that it will be in. So if there is an ev
 
 Fill in the data as follows:
 * `event:` The name of the event goes here as well as the date. The recommended format is: `Event Name - January 1st, 2000`
-* `where:` Where the event will take place. Vanue name and address are recommended.
+* `where:` Where the event will take place. Venue name and address are recommended.
 * `when:` Date and time
-* `description:` Descriptoin of the happenings of your event
+* `description:` Description of the happenings of your event
 * `link:` The website of your event (if applicable, this can be left blank and everything will be ok). This link must have http:// at the beginning if it is an external link.
 
 **Make sure the indentation is EXACTLY the same as the other proposals in the area. If it's not the jekyll build WILL fail.**
@@ -326,7 +335,7 @@ Find the year that you want to update and copy the code below:
   date:
   status:
 ```
-and paste it in the correct years `accomplishments:` section IN THE CHRONOLOGICAL ORDER that it will be in. So if there is a accomplishment that happened/is happening before the one you are inputting, make sure it is higher up than yours. If there is an accomplishment that happened/is happening AFTER yours, make sure it is after yours. If you don't have exact dates, just do your best to estimate.
+and paste it in the correct year's `accomplishments:` section IN THE CHRONOLOGICAL ORDER that it will be in. So if there is a accomplishment that happened/is happening before the one you are inputting, make sure it is higher up than yours. If there is an accomplishment that happened/is happening AFTER yours, make sure it is after yours. If you don't have exact dates, just do your best to estimate.
 
 Fill in the data as follows:
 * `name:` The name of the accomplishment. Try to keep it short, a sentence or two at most.
@@ -360,7 +369,7 @@ Find the category that best describes your business/service and copy the code be
 and paste it in the correct category under the `merchants:` section.
 
 Fill in the data as follows:
-* `name:` The name of the the business/service.
+* `name:` The name of the business/service.
 * `url:` The external url of the business/service. This link must have http:// (or https://) at the beginning if it is an external link.
 
 **Make sure the indentation is EXACTLY the same as the other proposals in the area. If it's not the jekyll build WILL fail.**
@@ -378,7 +387,7 @@ You're all done. Submit a PR and wait for it to be reviewed and merged. Be sure 
 ## 12.0 How to add a question to the FAQ
 
 ### 12.1 Copy/Paste Code
-Navigate to the `/get-started/faq` folder and open the `index.md` file. Inside you will see HTML code, but you will see it is very repetetive.
+Navigate to the `/get-started/faq` folder and open the `index.md` file. Inside you will see HTML code, but you will see it is very repetitive.
 
 Copy the code below:
 ```
@@ -394,7 +403,7 @@ CHANGE ANSWER
 
 </div>
 ```
-And paste it at the very bottom of the file. Literally underneath everything else.
+And paste it at the very bottom of the file (literally underneath everything else).
 
 Now we're going to change just a couple of things. Find the section with:
 ```

